src/lib/browse/layers/areas/CensusOutputAreas.svelte

```diff
@@ -41,26 +37,6 @@
   }
   let state = customUrlState(name, defaultState, stringify, parse);
 
-  // Mutually exclusive, like a radio button. We need these for checkboxes to work
```

Comment:
Big change to this layer to use radio buttons. Probably should've been like this from the start